### ISSUE TITLE
Please release new version to correctly work with NodeJS v12

### ISSUE THREAD

**Issue by smonv** [nestjsx/nest-emitter#28]
After upgrade to NodeJS v12, nest-emitter no longer buildable.

```
node_modules/nest-emitter/nest-emitter.module.d.ts(3,35): error TS2694: Namespace 'NodeJS' has no exported member 'Events'.
```

#13 fixed this issue and already merged.

Please release a new version.

**Comment by vidi42** [comment]
+1

**Comment by fr1sk** [comment]
+1

**Comment by dev-marceloborba** [comment]
+1

**Comment by shekohex** [comment]
published [`v1.1.1`](https://www.npmjs.com/package/nest-emitter)

**Comment by smonv** [comment]
@shekohex Thank you for the release.
